python-sdk/tests/databases/test_bigquery.py

```diff
@@ -155,6 +155,95 @@ def test_load_pandas_dataframe_to_table(database_table_fixture):
     assert rows[1] == (2,)
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "database_table_fixture",
+    [
+        {"database": Database.BIGQUERY},
+    ],
+    indirect=True,
+    ids=["bigquery"],
+)
+def test_load_pandas_dataframe_to_table_with_append(database_table_fixture):
+    """Load Pandas Dataframe to a SQL table with append strategy"""
+    database, table = database_table_fixture
+
+    pandas_dataframe = pd.DataFrame(data={"id": [1, 2]})
+    database.load_pandas_dataframe_to_table(
+        source_dataframe=pandas_dataframe,
+        target_table=table,
+        if_exists="append",
+    )
+
+    statement = f"SELECT * FROM {database.get_table_qualified_name(table)};"
+    response = database.run_sql(statement)
+
+    rows = response.fetchall()
+    assert len(rows) == 2
+    assert rows[0] == (1,)
+    assert rows[1] == (2,)
+
+    database.load_pandas_dataframe_to_table(
+        source_dataframe=pandas_dataframe,
+        target_table=table,
+        if_exists="append",
+    )
+
+    statement = f"SELECT * FROM {database.get_table_qualified_name(table)};"
+    response = database.run_sql(statement)
+    rows = response.fetchall()
+    assert len(rows) == 4
+    assert rows[0] == (1,)
+    assert rows[1] == (2,)
+    assert rows[2] == (1,)
+    assert rows[3] == (2,)
+
+    database.drop_table(table)
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "database_table_fixture",
+    [
+        {"database": Database.BIGQUERY},
+    ],
+    indirect=True,
+    ids=["bigquery"],
+)
+def test_load_pandas_dataframe_to_table_with_replace(database_table_fixture):
```

Comment:
Can we not parameterise the test for all databases instead of writing the same test multiple times across databases?

Reply:
yeah, we can do but a couple of other tests in a similar fashion (i.e not parameterise all DB together) I have moved tests to [test_all_databases.py](https://github.com/astronomer/astro-sdk/pull/1293/commits/470f41daa13532fcd705c077dcdb887ac8fe6ada) 470f41daa13532fcd705c077dcdb887ac8fe6ada